src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/OrganisationExternalControllerFunctionalTest.java

```diff
@@ -0,0 +1,199 @@
+package uk.gov.hmcts.reform.professionalapi;
+
+import lombok.extern.slf4j.Slf4j;
+import net.thucydides.core.annotations.WithTag;
+import net.thucydides.core.annotations.WithTags;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.hmcts.reform.professionalapi.controller.request.ContactInformationCreationRequest;
+import uk.gov.hmcts.reform.professionalapi.controller.request.NewUserCreationRequest;
+import uk.gov.hmcts.reform.professionalapi.controller.request.OrganisationCreationRequest;
+import uk.gov.hmcts.reform.professionalapi.domain.UserProfileUpdatedData;
+import uk.gov.hmcts.reform.professionalapi.util.FeatureToggleConditionExtension;
+import uk.gov.hmcts.reform.professionalapi.util.ToggleEnable;
+import uk.gov.hmcts.reform.professionalapi.util.serenity5.SerenityTest;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static org.apache.commons.lang3.ObjectUtils.isEmpty;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.codehaus.groovy.runtime.InvokerHelper.asList;
+import static org.springframework.http.HttpStatus.CREATED;
+import static org.springframework.http.HttpStatus.OK;
+import static uk.gov.hmcts.reform.professionalapi.client.ProfessionalApiClient.createContactInformationCreationRequests;
+import static uk.gov.hmcts.reform.professionalapi.client.ProfessionalApiClient.createOrganisationRequest;
+import static uk.gov.hmcts.reform.professionalapi.controller.constants.IdamStatus.SUSPENDED;
+import static uk.gov.hmcts.reform.professionalapi.controller.request.UserCreationRequest.aUserCreationRequest;
+
+@SerenityTest
+@SpringBootTest
+@WithTags({@WithTag("testType:Functional")})
+@Slf4j
+@SuppressWarnings("unchecked")
+public class OrganisationExternalControllerFunctionalTest extends AuthorizationFunctionalTest {
```

Comment:
Please use `ProfessionalExternalUserFunctionalTest` class for your new Tests as this is where all External API tests go

Reply:
resolved